server/services/participants.js

```diff
@@ -420,6 +421,17 @@ const getParticipants = async (
     total: Number(await table.count(criteria || {})),
   };
 
+  // HCAP:1030: Get participants post-hire statuses
+  participants = await Promise.all(
```

Comment:
What are your thoughts around adding post hire statuses to the response  body of the main getParticipants call?

Reply:
in participant view we have to show graduation status ....